pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -854,19 +857,29 @@ var (
 	}
 
 	AWSGroup = apiextensionsv1beta1.JSONSchemaProps{
-		Description: "",
+		Type: "object",
 		Properties: map[string]apiextensionsv1beta1.JSONSchemaProps{
-			"SecurityGroupsIds": {
+			"securityGroupsIds": {
```

Comment:
Is there any meaningful potential impact with the case change here or are they seen as the same thing when upgrading schemas?